less/index.less

```diff
@@ -4,173 +4,363 @@
+      margin-bottom: 20px;
+    }
+
+    nav {
+      margin-top: 40px;
+      width: 100%;
+      justify-content: space-between;
+      display: flex;
+
+      @media (max-width: 400px) {
+        .center-stuff();
+        flex-direction: column;
+
+        a {
+          width: 100%;
+          padding-bottom: 20px;
+          border-top: 1px solid @color-scheme-primary;
+          .center-stuff();
+        }
+      }
+
+      a {
+        text-decoration: none;
+        color: @color-scheme-secondary;
+      }
+
+      a:hover {
+        text-decoration: underline;
+        color: black;
+        border: 1px * 3 solid black;
+      }
+    }
+
+    .logo {
+      align-self: center;
+      margin: 28px 0 0 91px;
+
+      @media (max-width: 768px) {
+        align-self: center;
+        margin: 28px 0 0 0;
+      }
+      @media (max-width: 400px) {
+        align-self: center;
+        margin: 28px 0 0 0;
+      }
+    }
+  }
 }
 
 /* Call To Action (CTA) Styles */
 
 .cta {
-    display: flex;
-    margin-top: 72px;
-    justify-content: flex-end;
-}
+  width: 100%;
+  display: flex;
+  margin-top: 72px;
+
+  // Tablet Styling ===============
+  @media (max-width: 768px) {
+    width: 100%;
+    margin: 0 auto;
+    .center-stuff;
+    flex-direction: column;
+  }
 
-.cta .cta-text {
-    font-size: 72px;
+  .cta-text {
+    width: 100%;
     text-align: center;
-    letter-spacing: 5px;
     display: flex;
     justify-content: center;
     flex-flow: column;
     align-items: center;
-    margin: 0 82px 0 0;
-}
 
-.cta .cta-text button {
-    border: 1px double black;
-    font-size: 16px;
-    background: white;
-    width: 170px;
-    height: 32px;
-    margin-top: 10px;
-    cursor: pointer;
-}
+    h1 {
+      width: 100%;
+      font-size: 7.6vmin;
+    }
+
+    button {
+      border: 1px double black;
+      font-size: 1.6rem;
+      background: white;
+      width: 170px;
+      height: 32px;
+      margin-top: 10px;
+      cursor: pointer;
+
+      // Tablet Styling ===============
+      @media (max-width: 768px) {
+        margin-bottom: 14px;
+      }
+    }
 
-.cta .cta-text button:hover {
-    background: black;
-    color: white;
+    button:hover {
+      background: black;
+      color: white;
+    }
+  }
+
+  img {
+    @media (max-width: 768px) {
+      max-width: 456px;
+      width: 100%;
+    }
+  }
 }
 
 /* Main Content Styles */
 
 .main-content {
-   border-top: 2px solid black; 
-   border-bottom: 2px solid black;
-   margin: 75px 0 35px 0;
-   padding: 35px 0;
-   font-size: 16px;
-   line-height: 18px;
-}
+  border-top: 2px solid black;
+  border-bottom: 2px solid black;
+  margin: 75px 0 35px 0;
+  padding: 35px 0;
+  font-size: 1.6rem;
+  line-height: 18px;
+  color: @color-scheme-primary;
 
-.main-content .top-content {
-    display: flex;
-    justify-content: space-around;
-}
+  h4 {
+    margin-top: 10px;
+  }
+
+  p {
+    padding: 5px;
+  }
 
-.main-content .middle-img {
+  .top-content {
```

Comment:
Here's some food for thought:

So first, great job using a single mixin for both `.top-content` and `.bottom-content`. But let's take this further, theoretically. I'll share one interesting idea another student employed:

>`//.flex-mix(@dir: column; @align: center); || .flex-mix(@just: center;) || etc`
>`.flex-mix(@just: flex-start; @align: stretch; @dir: row; @wrap: nowrap) {`

>>  `display: flex;`
>  `justify-content: @just;`
>  `align-items: @align;`
>  `flex-flow: @dir @wrap;`

>`}`

Sorry if it comes out weird, I'm having trouble getting github to style it correctly. Anyway, this is kind of a neat idea. I'm showing you this to get you thinking about how you might employ some parametric mixins with some default settings as well. (So with this, you never have to give all or any values into the mixin.) I'm not saying this is the best application, but it shows you some possibilities to get your mind thinking about parametric mixins and default values.

Reply:
Yeah Joseph and I went over what we were calling The Flex Machine for like 30 minutes yesterday, it's a parametric mixin that lets you pass values to all those typical layout stylings.  Rather than the || operator he just used multiple declarations in the parentheses.  It opened my eyes as to how subtle DRY can get.  Then I asked Joseph "this is cool to understand this stuff under the hood but doesn't all this get irrelevant when we get to Grid and Bootstrap?"  to which he replied "yep."

Still good to wrap one's head around.

Reply:
To clarify, Joseph built it, he just explained to me how it worked.  It wouldn't have occured to me to build that.